FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API offers methods and events for text finding and navigation
+within a WebView2 control. It enables developers to programmatically initiate find
+operations, navigate find results, suppress default UI, and customize find configurations
+like query and find direction. It also tracks the status of operations, indicating
+completion, match count changes, and match index changes.
+
+## Examples
+
+
+#### Description
+
+To initiate a find operation in a WebView2 control, use the `StartFind` method.
+This method allows setting the find term and find parameters via the
+`ICoreWebView2FindConfiguration` interface. Only one find session can be active per
+webview environment. Starting another with the same configuration will adjust
+the active match index based on the selected Find Direction.
+### Create/Specify a Find Configuration
+#### WIN32 C++
+
+```cpp
+
+//! [InitializeFindConfiguration]
+wil::com_ptr<ICoreWebView2FindConfiguration> AppWindow::InitializeFindConfiguration(const std::wstring& findTerm)
+{
+    // Query for the ICoreWebView2Environment5 interface.
+    auto webView2Environment5 = m_webViewEnvironment.try_query<ICoreWebView2Environment5>();
+    CHECK_FEATURE_RETURN(webView2Environment5);
+
+    // Initialize find configuration/settings
+    wil::com_ptr<ICoreWebView2FindConfiguration> findConfiguration;
+    CHECK_FAILURE(webView2Environment5->CreateFindConfiguration(&findConfiguration));
+    CHECK_FAILURE(findConfiguration->put_FindTerm(findTerm.c_str()));
+    CHECK_FAILURE(findConfiguration->put_IsCaseSensitive(false));
+    CHECK_FAILURE(findConfiguration->put_ShouldMatchWord(false));
+    CHECK_FAILURE(findConfiguration->put_FindDirection(COREWEBVIEW2_FIND_DIRECTION_FORWARD));
+
+    // Query for the ICoreWebView2_17 interface to access the Find feature.
+    auto webView2_17 = m_webView.try_query<ICoreWebView2_17>();
+    CHECK_FEATURE_RETURN(webView2_17);
+
+    return findConfiguration;
+}
+//! [InitializeFindConfiguration]
+```
+
+```cpp
+//! [ExecuteFindWithDefaultUI]
+bool AppWindow::ConfigureAndExecuteFind(const std::wstring& findTerm) 
+{
+    auto findConfiguration = InitializeFindConfiguration(findTerm);
+    if (!findConfiguration)
+    {
+        return false;
+    }
+    // Query for the ICoreWebView2_17 interface to access the Find feature.
+    auto webView2_17 = m_webView.try_query<ICoreWebView2_17>();
+    CHECK_FEATURE_RETURN(webView2_17);
+
+    // Get the Find interface.
+    wil::com_ptr<ICoreWebView2Find> webView2Find;
+    CHECK_FAILURE(webView2_17->get_Find(&webView2Find));
+
+    // By default Find will use the default UI and highlight all matches. If you want different behavior
+    // you can change the SuppressDefaultDialog and ShouldHighlightAllMatches properties here.
+
+    // Start the find operation with a callback for completion.
+    CHECK_FAILURE(webView2Find->StartFind(
```

Comment:
The IDL calls this `StartFindAsync`, but I think "Start" and "Async" are redundant, we should go with one or the other

Reply:
Please fix:
 * COM: Start
 * .NET/WinrT: StartAsync